src/styles/utilities/display.ts

```diff
@@ -20,15 +19,35 @@ export default {
         display: 'none',
     },
 
+    /**
+     * Web-only style.
+     */
     dInline: {
-        display: 'inline',
+        // NOTE: asserting "display" to a valid type, because isn't possible to augment "display".
```

Comment:
typo (same as below)
```suggestion
        // NOTE: asserting "display" to a valid type, because it isn't possible to augment "display".
```